package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "azuredatastudio",
   "version": "1.46.0",
-  "distro": "ddc1ad96839c9128a60123f438335d3c8b2d3480",
+  "distro": "7f4613d9a9266732a1d3a616e3d4112495083a71",
```

Comment:
Updating the distro hash to resolve distro merge conflicts is the only change I made to this PR.